immunisation_subs.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import requests
+import sys
+
+
+def create_sub(url, token, data):
+    headers = {
+        'Authorization': "Token " + token,
+        'Content-Type': "application/json"
+    }
+    return requests.post('%ssubscriptions/' % url, headers=headers, json=data)
```

Comment:
I think it might be better here to use a session with retries, in case there are connection errors https://github.com/praekelt/hellomama-registration/blob/226f5d7d824f32383a23efec8e44912a6aa04b26/hellomama_registration/utils.py#L10-L14